python/cudf/dataframe/series.py

```diff
@@ -1833,11 +1833,32 @@ def quantile(self, q=0.5, interpolation='linear', exact=True,
         DataFrame
 
         """
+
+        if isinstance(q, (int, float)) and not (0 <= q <= 1):
```

Comment:
This should be more generalized to isinstance(q, numbers.Number)

Reply:
Done